src/system_interfaces/output_loop.ts

```diff
@@ -0,0 +1,65 @@
+import { invoke } from '@tauri-apps/api/core';
+import { isTauri } from './util';
+
+export const outputLoopSupported = isTauri;
+
+export const startOutputLoop = isTauri
+  ? tauriStartOutputLoop
+  : webStartOutputLoop;
+export const stopOutputLoop = isTauri
+  ? tauriStopOutputLoop
+  : webStopOutputLoop;
+export const rebuildOutputLoops = isTauri
+  ? tauriRebuildOutputLoops
+  : webRebuildOutputLoops;
+
+async function tauriStartOutputLoop(
+  outputId: bigint,
+  outputType: 'serial' | 'sacn' | 'wled',
+  options: {
+    universe?: number;
+    ipAddress?: string;
+    targetFps?: number;
+  } = {},
```

Comment:
Lets make an interface that is a union type of the different types of outputs. Thus it will require the sACN state to be passed in when required.